utility/Vector.cpp

```diff
@@ -17,6 +17,12 @@ Vector3 Vector3::CrossProduct(const Vector3& a, const Vector3& b)
                    a.X * b.Y - a.Y * b.X);
 }
 
+float Vector3::GetDistance(const Vector3& a, const Vector3& b)
```

Comment:
Can you change this to not be static, taking only one `Vector3`?  Also, would you mind searching for where this calculation is done elsewhere and updating the code to use this function?

Reply:
Made the change to not static, not sure if its right. I couldn't find another use for distance calc between 3d vectors.